easybuild/easyconfigs/t/Triangle/Triangle-1.6-GCCcore-13.2.0.eb

```
@@ -0,0 +1,39 @@
+easyblock = 'MakeCp'
+
+name = 'Triangle'
+version = '1.6'
+
+homepage = 'https://www.cs.cmu.edu/~quake/triangle.html'
+description = """Triangle generates exact Delaunay triangulations, constrained Delaunay triangulations,
+ conforming Delaunay triangulations, Voronoi diagrams, and high-quality triangular meshes.
+ The latter can be generated with no small or large angles,
+ and are thus suitable for finite element analysis."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.netlib.org/voronoi/']
+sources = ['%(namelower)s.zip']
+checksums = [
+    '1766327add038495fa3499e9b7cc642179229750f7201b94f8e1b7bee76f8480',  # triangle.zip
+    '38fc1395c2392f627068b669275c30da0c25b1a6db8ed8b80d6bf05a98971568',  # Triangle-1.6_makefile.patch
+]
+
+patches = ['%(name)s-%(version)s_makefile.patch']
+
+builddependencies = [('binutils', '2.40')]
+
+buildopts = 'triangle trilibrary'
+
+files_to_copy = [
+    (['triangle', 'tricall'], 'bin'),
+    (['triangle.h'], 'include'),
+    (['libtriangle.a'], 'lib'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/triangle', 'bin/tricall', 'include/triangle.h', 'lib/libtriangle.a'],
+    'dirs': []
+}
+
```

Let's add sanity check commands here as well:
```suggestion

sanity_check_commands = [
    'triangle -h',
    'tricall',
]

```